install/certmanager/041-Certificate.yaml

```diff
@@ -13,4 +13,6 @@ spec:
     - strimzi-drain-cleaner.strimzi-drain-cleaner
     - strimzi-drain-cleaner
   issuerRef:
+    group: cert-manager.io
+    kind: Issuer
```

Comment:
As for the other PR, these are default values so we can leave them out imho.

Reply:
which other PR?
this is in the released files so it should be included in the install folder as well or?